docs/UserGuide.md

```diff
@@ -125,42 +118,7 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd patient in the patient book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
```

Comment:
I think for this part `deletePatient` should be before `list` and `find` since that is the order in the command summary table.
But maybe it would be better to reorder the command summary table instead since `deletePatient` requires you to use the `list` command first.

and also should change the `delete 2` and the `delete 1` in the examples to `deletePatient 2` and etc.